### PR TITLE
change entity and remove useless entity

### PR DESCRIPTION
**Pull request by haboy52581**

### CODE REVIEW

File: web/src/main/java/com/futurewei/alcor/web/entity/subnet/SubnetEntity.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
 @Data
 public class SubnetEntity extends CustomerResource {
 
-    @JsonProperty("network_id")
+    @JsonProperty("vpc_id")
```
Comment (xieus) [inline web/src/main/java/com/futurewei/alcor/web/entity/subnet/SubnetEntity.java:31]: Please revert this change. Externally we use "network_id" to be OpenStack compatible.

File: web/src/main/java/com/futurewei/alcor/web/entity/dataplane/NetworkConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import com.futurewei.alcor.schema.Common;
+import com.futurewei.alcor.web.entity.securitygroup.SecurityGroupEntity;
+import com.futurewei.alcor.web.entity.subnet.SubnetEntity;
+import com.futurewei.alcor.web.entity.vpc.VpcEntity;
+import lombok.Data;
+
+@Data
+public class NetworkConfiguration {
+
+  private Common.ResourceType rsType;
```
Comment (xieus) [inline web/src/main/java/com/futurewei/alcor/web/entity/dataplane/NetworkConfiguration.java:13]: What are these? As commented in PR https://github.com/futurewei-cloud/alcor/pull/234/files and discussed offline, we don't need this in the north bound configuration.

File: web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntityNB.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import com.futurewei.alcor.web.entity.vpc.VpcEntity;
+import lombok.Data;
+
+import java.util.Set;
+
+@Data
+public class InternalPortEntityNB extends PortEntity {
+
+  @JsonProperty("neighbor_host_ip")
+  private String neighborIp;
```
Comment (xieus) [inline web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntityNB.java:15]: So is it assumed that there is only one neighbor?

File: web/src/main/java/com/futurewei/alcor/web/entity/dataplane/NetworkConfiguration.java (hunk 2 above)
Comment (haboy52581) [reply web/src/main/java/com/futurewei/alcor/web/entity/dataplane/NetworkConfiguration.java:13]: changed in dcede47

File: web/src/main/java/com/futurewei/alcor/web/entity/subnet/SubnetEntity.java (hunk 1 above)
Comment (haboy52581) [reply web/src/main/java/com/futurewei/alcor/web/entity/subnet/SubnetEntity.java:31]: changed in dcede47

File: web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntityNB.java (hunk 3 above)
Comment (haboy52581) [reply web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntityNB.java:15]: changed in dcede47, as discussed yes for each port

**Comment by haboy52581** [comment]
would continue working on #234

**Comment by haboy52581** [comment]
deleted dev/work-0 since moving the change to dev/add-entity and close this PR

**Comment by xieus** [comment]
Duplicated of PR https://github.com/futurewei-cloud/alcor/pull/240.
